app/styles/_user.less

```diff
@@ -0,0 +1,10 @@
+// TODO: make these more specific. 
```

Comment:
Are these styles going to be used anywhere else?  If not just make this part of membership.less